libbeat/licenser/elastic_fetcher.go

```diff
@@ -107,6 +108,7 @@ type esClientMux struct {
 // at the end of the function call, if an error occur we return the error and will pick up the next client on the
 // next call. Not that we just round robin between hosts, any backoff strategy should be handled by
 // the consumer of this type.
+// nolint: unused // it's used on Linux
```

Comment:
```suggestion
//nolint:unused // it's used on Linux
```

are u sure this comment is working?